docs/src/main/asciidoc/infinispan-dev-services.adoc

```diff
@@ -0,0 +1,114 @@
+
+Available properties to customize the Infinispan DevService.
+
+include::{generated-dir}/config/quarkus-infinispan-client-config-group-dev-services-config.adoc[opts=optional, leveloffset=+1]
+
+When running the production version of the application, the Infinispan connection need to be configured as normal,
+so if you want to include a production database config in your `application.properties` and continue to use Dev Services
+we recommend that you use the `%prod.` profile to define your Infinispan settings.
+
+Dev Services for Infinispan relies on Docker to start the server.
+
+== Connecting to the running Infinispan Server
+
+You *don't need to configure anything in dev mode*.
+The server will be running in a random port.
+If you need a fixed port, then configure `quarkus.infinispan-client.devservices.port` property.
+
+[NOTE]
+====
+The running Infinispan Server has authentication enabled and a user with full admin role.
+The credentials of the user are *admin/password*.
+====
+
+=== Accessing to the Infinispan Server Console
+Infinispan Server provides a web console that can be accessed with a browser:
+
+. Open the xref:dev-ui.adoc[Dev UI]
+. You will see an *Infinispan Client* box. Click on  *Web Console* and enter the credentials above.
+
+If your environment does not support Docker, you will need to spin up an Infinispan Server manually, or connect to an already running server.
+
+== Overriding the Infinispan Server Image
+
+The extension is updated regularly and the Dev Services will start the latest final version image of Infinispan.
+Use `quarkus.infinispan-client.devservices.image-name` property to specify another image that fits your needs.
+
+==  Enabling / Disabling Dev Services for Infinispan
+
+Dev Services for Infinispan is automatically enabled unless:
+
+- `quarkus.infinispan-client.devservices.enabled` is set to `false`
+- the `quarkus.infinispan-client.server-list` is configured
+
+Dev Services for Infinispan relies on Docker to start the broker.
+If your environment does not support Docker, you will need to start the broker manually, or connect to an already running broker.
+You can configure the broker address using `quarkus.infinispan-client.server-list`.
+
+== Cross Site Replication
+If you want run the Infinispan Server container with Cross Site Replication configuration, the required configuration properties
+are the following:
+
+[source,properties]
+----
+quarkus.infinispan-client.devservices.site=NYC <1>
+quarkus.infinispan-client.devservices.mcast-port=46666 <2>
+----
+
+<1> Provides a site name for your Infinispan cluster
+<2> Mcast port need to be different between sites if you are running more than one site.
+
+
+== Tracing with OpenTelemetry
+
+Infinispan traces the server using OpenTelemetry. Enable tracing setting `quarkus.infinispan-client.devservices.tracing.enabled` to true.
```

Comment:
The sentence `Infinispan traces the server using OpenTelemetry` is not really correct. I assume you mean `instrument` instead of `trace`, but even if you change that, I still don't think the sentence is great. Perhaps something like:

`Infinispan supports instrumentation of the server via OpenTelemetry`